packages/Uploader/src/Uploader.js

```diff
@@ -61,6 +63,10 @@ const propTypes = {
     you can pass an array of header objects.
   */
   headers: PropTypes.arrayOf(PropTypes.object),
+  /**
+    This callback fires every time a file has been processed.
+  */
+  onProcess: PropTypes.func,
```

Comment:
I think `onProcessed` is more clear that this is happening _after_ the file has been processed